.github/workflows/main_ci.yml

```diff
@@ -23,11 +20,8 @@ jobs:
     strategy:
       matrix:
         path:
-          - 'include'
-          - 'src'
-          - 'test'
-          - 'cmd'
-          - 'lib'
+          - '.'
+          - 'cmd/interop'
```

Comment:
isn't `cmd/interop` a subset of `.`

Reply:
No, because the `include-regex` limits makes each of these only apply to `src|include|lib|test`.